test/utils.spec.ts

```diff
@@ -1,13 +1,113 @@
+import * as THREE from "three";
+import View3DError from "~/View3DError";
 import {
   getCanvas,
   range,
   toRadian,
   clamp,
   findIndex,
+  getElement,
+  mix,
+  circulate,
+  merge,
+  getBoxPoints,
+  toPowerOfTwo,
 } from "~/utils";
 import * as ERROR from "~/consts/error";
+import { cleanup, createSandbox } from "./test-utils";
 
 describe("Utils", () => {
+  describe("getElement", () => {
+    let wrapper: HTMLElement;
+
+    beforeEach(() => {
+      wrapper = createSandbox("#wrapper");
+    });
+
+    afterEach(() => {
+      cleanup();
+    });
+
+    it("should search from the document with selector if the parent element is not given", () => {
+      // Given
+      const findingEl = document.createElement("div");
+      findingEl.id = "target";
+
+      // When
+      document.body.appendChild(findingEl);
+
+      // Then
+      try {
+        expect(() => getElement("#target")).not.to.throw();
```

Comment:
It looks like a meaningless comparison?

Reply:
Looks like so, I guess I wrote that line habitually.